packages/dd-trace/test/instrumenter.spec.js

```diff
@@ -352,4 +353,60 @@ describe('Instrumenter', () => {
       })
     })
   })
+
+  describe('with plugins disabled via disabledPlugins configuration option', () => {
```

Comment:
This should be reworded to make it clearer that it's an environment variable. Currently it leads to believe there is a `disabledPlugins` option in code.

Reply:
updated